### PR TITLE
CORE-986 More simulations

### PR DESCRIPTION
**Pull request by ys-pyrofex**
## Overview
We need more simulations in perf-harness to track current performance and progress of our optimization efforts.

### Which JIRA issue does this PR relate to? 
https://rchain.atlassian.net/browse/CORE-986

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,19 @@ builder/envs/**
 *.log
 templater/results/**
 
+# Maven Output #
+#################
+target
+
+# Intellij IDEA Project Files
```
Comment (dzajkowski) [inline .gitignore:11]: shouldn't this be in a global git ignore file on a devs machine? if not - we need to add the git ignores for emacs, eclipse, sublime and so on

File: templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import io.gatling.core.Predef.{Simulation, atOnceUsers, scenario}
+import io.gatling.core.Predef._
+
+import scala.concurrent.duration._
+import scala.io.Source
+
+/*
+ We expect to reach 40k comm events per second one day
+ */
+class CommEvents40kSimulation extends Simulation {
```
Comment (dzajkowski) [inline templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala:15]: have you made sure that these new simulations don't interfere with how the harness is used for performance tests?

Comment (lukasz-golebiewski) [reply templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala:15]: Looks like they got executed even when a run with a custom contract was desired:
http://stress-docker.pyr8.io:8080/rchain/rchain-perf-harness/213

File: .gitignore (hunk 1 above)
Comment (ys-pyrofex) [reply .gitignore:11]: I just copied these lines from the main project, assuming that if it's ok for the main project, then it's ok for perf-harness too. Don't know ignores for other editors, so decided to copy-paste section "# Actual code editors" from the main project.

File: templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala (hunk 2 above)
Comment (ys-pyrofex) [reply templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala:15]: Thanks for noticing, they're really inferred separate scenarios (custom commit & custom contract). Fixed.

**Pull request merged**